recipes/trinity/date.2011_11_26/meta.yaml

```diff
@@ -0,0 +1,40 @@
+about:
+    home: 'https://github.com/trinityrnaseq/trinityrnaseq'
+    license: "https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt"
+    summary: "Trinity assembles transcript sequences from Illumina RNA-Seq data"
+build:
+  number: 5
+  string: ncurses{{CONDA_NCURSES}}_{{PKG_BUILDNUM}}
+package:
+  name: trinity
+  version: 'date.2011_11_26'
+source:
+  fn: trinityrnaseq_r2011-11-26.tgz
+  md5: 3a4c4691908d37042ca63ca469c75fd3
+  url: http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz
+  patches:
+    - RunButterfly.cc.patch # this file is missing a header include
+    - Trinity.pl.patch # Trinity requires exactly Java 1.6, but works fine >1.6, so adjust to be more permissive
+    - samtools_makefile.patch # patch samtools Makefile due to missing curses.h
+requirements:
+  build:
+    - perl-threaded
+    - gcc
+    - java-jdk >=6
+    - bzip2
+    - zlib
+    - ncurses {{CONDA_NCURSES}}*
+  run:
+    - perl-threaded
+    - libgcc
+    - java-jdk >=6
+    - bzip2
+    - zlib
+    - ncurses {{CONDA_NCURSES}}*
+extra:
+  notes: "Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)"
+test:
+    commands:
+        # This version of Trinity does not have a quick function to exit 0
+        # So at least check that we can compile the file
+        - "$PREFIX/bin/Trinity-test &> /dev/null"
```

Comment:
Related to this, is there a way we can include Docker build information in each PR in this repo? It would be nice if there were an automated way to know if we've broken something, so @bgruening doesn't have to manually write a comment. A comment bot, or maybe something optional in the test panel?